Generate docs (#2356)
kyma-project/cli: docs/user/gen-docs/01-70-kyma_alpha_hana_map.md

# kyma alpha hana map

> Map the Hana instance to the Kyma cluster

Reviewer: ```suggestion
Maps an SAP HANA instance to the Kyma cluster.
```

## Synopsis

Use this command to map the Hana instance to the Kyma cluster

```bash
kyma alpha hana map [flags]
```

## Flags

```bash
      --credentials-path string   Path to the credentials json file
      --hana-id string            Hana instance ID
```

## Flags inherited from parent commands

```bash
  -h, --help                    Help for the command
      --kubeconfig string       Path to the Kyma kubeconfig file
      --show-extensions-error   Print possible error when fetching extensions failed
```

## See also

> * [kyma alpha hana](kyma_alpha_hana.md) - Manage a Hana instance on the Kyma platform

Reviewer: ```suggestion
* [kyma alpha hana](kyma_alpha_hana.md) - Manages an SAP HANA instance on SAP BTP.
```
